common/sgxcclib/common-sgxcclib.h

```diff
@@ -29,6 +30,25 @@ int sgxcc_get_remote_attestation_report(enclave_id_t eid,
 int sgxcc_get_pk(enclave_id_t eid, ec256_public_t* pubkey);
 int sgxcc_get_egid(unsigned int* p_egid);
 
+#define ERROR_CHECK_RET(func_name)                             \
```

Comment:
> . you refer to ECC_ATTESTATION_TYPE ? (or put differently, you are saying that tlcc is using the same attestation type as defined (defined by ECC_ATTESTATION_TYPE env variable)?
This would of course also lead to a naming suggestion to turn this into FPC_ATTESTATION_TYPE ...

fixed -- better doing it now than having issues later.

> standardizing and simplifying via macros error handling seems a good idea, but i have no idea on how to interpret from the three names the semantics. Also do we need func_name? Note the change i've done to logging which should give you context (file/line)?

renamed and made somewhat more consistent